Sources/DependenciesMacrosPlugin/Support.swift

```diff
@@ -96,8 +96,8 @@ extension InitializerClauseSyntax {
             node: self.value,
             message: MacroExpansionWarningMessage(
               """
-              Do not use 'unimplemented' with '@\(attribute.attributeName)'; it is a replacement and \
-              implements the same runtime functionality as 'unimplemented' at compile time
+              Do not use 'unimplemented' with '@\(attribute.attributeName)'; the '@\(attribute.attributeName)' macro \
+              provides an 'unimplemented' default value for you.
```

Comment:
Thoughts on a lil more workshopping?

```suggestion
              Do not use 'unimplemented' with '@\(attribute.attributeName)'; the \
              '@\(attribute.attributeName)' macro already includes the behavior of \
              'unimplemented'.
```